app/views/toggle/index.html.erb

```diff
@@ -149,9 +176,19 @@
     Temporary
   </div>
   <div class="">
-    <% @facade.temporary_toggles.each do |toggle| %>
-      <%= render 'toggle/toggle_row', toggle: toggle %>
-    <% end %>
+    <table class="toggle-table">
```

Comment:
Display temporary toggles in a table for improved layout, now that the updated at is displaying as well.